2.0/m_rehashsslsignal.cpp

```diff
@@ -50,7 +50,10 @@ class ModuleRehashSSLSignal : public Module
 		if (!signaled)
 			return;
 
-		ServerInstance->Logs->Log("m_rehashsslsignal", DEFAULT, "Got SIGUSR1, reloading SSL credentials");
+		std::string m = "Got SIGUSR1, reloading SSL credentials";
+		ServerInstance->SNO->WriteGlobalSno('a', m);
+		
```

Comment:
(Nitpicking here but) this line break might look better between the Log line and `const std::string str...`
ie. keep the three lines dealing with the message together.